README.md

```diff
@@ -1,16 +1,105 @@
 ![](https://github.com/projectriff/charts/workflows/CI/badge.svg)
 
-# projectriff Helm charts
+# projectriff Helm charts (and uncharts)
 
-Helm charts to install Istio and riff.
+Helm charts (and uncharts) to install Istio and riff.
 
-## Install
+## Install (kapp)
 
 ### Prerequisites
 
 - a running kubernetes cluster (1.14+)
-- kubectl (1.14+)
-- helm (2.13+)
+- [kubectl](https://kubectl.docs.kubernetes.io) (1.14+)
+- [kapp](https://get-kapp.io) (0.14+)
+- [ytt](https://get-ytt.io) (0.14+)
+
+### Steps
+
+1. Define riff version
+
+   ```sh
+   riff_version=0.5.0-snapshot
+   ```
+
+1. Install riff Build (and dependencies)
+   
+   ```sh
+   kapp deploy -a cert-manager -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/cert-manager.yaml
+   kapp deploy -a kpack -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/kpack.yaml
+   kapp deploy -a riff-builders -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/riff-builders.yaml
+   kapp deploy -a riff-build -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/riff-build.yaml
+   ```
+
+1. Optionally Install Istio (required for the Knative runtime)
+   
+   ```sh
+   kapp deploy -a istio -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/istio.yaml
+   ```
+   
+   If your cluster does not support LoadBalancer services (most hosted clusters do, and local clusters do not), then you'll need to convert the ingress service to a NodePort.
+   
+   ```sh
+   ytt -f https://storage.googleapis.com/projectriff/charts/uncharted/${riff_version}/istio.yaml -f https://storage.googleapis.com/projectriff/charts/overlays/service-nodeport.yaml --file-mark istio.yaml:type=yaml-plain | kapp deploy -a istio -f -
```

Comment:
instead of requiring ytt could we just replace this with:

```
kubectl patch svc -n istio-system istio-ingressgateway --type='json' -p '[{"op":"replace","path":"/spec/type","value":"NodePort"}]'
```

Reply:
it's better for resources managed by kapp to be managed by kapp. It's also a good way to get experience with updates.

Also, the patch requires more intimate knowledge of the istio install. The ytt transform managed all LoadBalancers uniformly.

Reply:
plus kapp won't get past the LoadBalancer without an IP address, it keeps waiting forever